src/IssueViz.Security/UI/SharedResources.xaml

```diff
@@ -0,0 +1,187 @@
+                                    <Path x:Name="SortArrow" 
+                                          Visibility="Collapsed" 
+                                          Data="M 0,0 L 1,0 0.5,1 z" 
+                                          Stretch="Fill"
+                                          Width="8"
+                                          Height="6" 
+                                          Fill="{DynamicResource {x:Static vsShell:VsBrushes.DockTargetGlyphArrowKey}}" 
+                                          Margin="5,0,0,0"
+                                          VerticalAlignment="Center" 
+                                          RenderTransformOrigin="0.5, 0.4"/>
+                                </StackPanel>
+                                <Thumb x:Name="PART_LeftHeaderGripper" HorizontalAlignment="Left" Style="{StaticResource ColumnHeaderGripperStyle}"/>
+                                <Thumb x:Name="PART_RightHeaderGripper" HorizontalAlignment="Right" Style="{StaticResource ColumnHeaderGripperStyle}"/>
+                            </Grid>
+                        </Border>
+                    </Border>
+                    <ControlTemplate.Triggers>
+                        <Trigger Property="IsMouseOver" Value="True">
+                            <Setter Property="Foreground" Value="{DynamicResource {x:Static vsShell:VsBrushes.CommandBarTextHoverKey}}"/>
+                            <Setter TargetName="BackgroundBorder" Property="Background" Value="{DynamicResource {x:Static vsTheming:HeaderColors.MouseOverBrushKey}}"/>
+                        </Trigger>
+                        <Trigger Property="IsPressed" Value="True">
+                            <Setter Property="Foreground" Value="{DynamicResource {x:Static vsShell:VsBrushes.CommandBarTextSelectedKey}}"/>
+                            <Setter TargetName="BackgroundBorder" Property="Background" Value="{DynamicResource {x:Static vsTheming:HeaderColors.MouseDownBrushKey}}"/>
+                        </Trigger>
+                        <Trigger Property="SortDirection" Value="Ascending">
+                            <Setter TargetName="SortArrow" Property="Visibility" Value="Visible"/>
+                            <Setter TargetName="SortArrow" Property="RenderTransform">
+                                <Setter.Value>
+                                    <RotateTransform Angle="180"/>
+                                </Setter.Value>
+                            </Setter>
+                        </Trigger>
+                        <Trigger Property="SortDirection" Value="Descending">
+                            <Setter TargetName="SortArrow" Property="Visibility" Value="Visible"/>
+                        </Trigger>
+                    </ControlTemplate.Triggers>
+                </ControlTemplate>
+            </Setter.Value>
+        </Setter>
+    </Style>
+    <Style TargetType="DataGridRow" x:Key="BaseDataGridRow">
```

Comment:
Style for the shareable 'DataGridRow' behaviours e.g. active/inactive selection.